docs/src/main/sphinx/connector/alter-schema-limitation.fragment

```diff
@@ -0,0 +1,5 @@
+ALTER SCHEMA support
+^^^^^^^^^^^^^^^^^^^^
+
+The connector supports the :doc:`/sql/alter-schema` ``RENAME``
```

Comment:
No need to link here again since it already links in the list above.

The connector supports renaming a schema with ``ALTER SCHEMA RENAME``.